### PR TITLE
Added quiesce hooks for Cassandra

### PR DESCRIPTION
**Pull request by kurwang**

### CODE REVIEW

File: cassandra/README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ INSERT into offices(officeCode, city, phone, addressLine1, addressLine2, state,
 ('1','San Francisco','+1 650 219 4782','100 Market Street','Suite 300','CA','USA','94080','NA');
 ```
 Verify that the data was populated.
+<<<<<<< HEAD
```
Comment (dymurray) [inline cassandra/README.md:32]: Looks like you accidentally committed this from a rebase/merge.

File: cassandra/README.md [hunk 2]
```diff
@@ -41,45 +42,37 @@ We can now exit the shell by simply exiting.
...
 ```
-oc exec -it cassandra-0 -- nodetool getsstables classicmodels offices 1
-oc exec -it cassandra-0 -- nodetool drain
+ansible-playbook backup.yaml
 ```
-Now before we take a backup of cassandra, lets also verify that cassandra has stop
-listening for connections. Try to access the cqlsh again. The following command should be an 
+After taking the backup of cassandra, lets also verify that cassandra has stop
+listening for connections. This means writes have also stopped occurring temporarily. Try to access the cqlsh again. The following command should be an 
 error like the following below after executing the command.
```
Comment (dymurray) [inline cassandra/README.md:56]: So how do we bring Cassandra back into an operational state after the drain command? I think that we also need a post hook which will bring the cluster up is that correct?

File: cassandra/roles/install-cassandra/templates/cassandra-app.yaml.j2 [hunk 3]
```diff
@@ -14,6 +14,8 @@ spec:
       app: cassandra
   template:
     metadata:
+      annotations:
+        pre.hook.backup.velero.io/command: '["/bin/bash", "-c", "opt/cassandra/bin/nodetool drain"]'
```
Comment (dymurray) [inline cassandra/roles/install-cassandra/templates/cassandra-app.yaml.j2:18]: This looks good, see above comment about whether or not we need a post hook.

File: cassandra/README.md (hunk 1 above)
Comment (kurwang) [reply cassandra/README.md:32]: Deleted. Had an error when merging

File: cassandra/README.md (hunk 2 above)
Comment (kurwang) [reply cassandra/README.md]: Changed to flush that will disable writes during backup and enable writes again after backup.

**Pull request merged**